Content/Metaballs/GrayGooMetaballs.cs

```diff
@@ -0,0 +1,97 @@
+﻿using StarlightRiver.Content.Items.Magnet;
+using StarlightRiver.Core.Systems.LightingSystem;
+using StarlightRiver.Core.Systems.MetaballSystem;
+using System.Linq;
+using Terraria.Graphics.Effects;
+
+namespace StarlightRiver.Content.Metaballs
+{
+	internal class GrayGooMetaballs : MetaballActor
+	{
+		public override bool Active => Main.dust.Any(x => x.active && (x.type == DustType || x.type == DustType2));
```

Comment:
Property overrides should be placed after properties, same for all other overrides here